src/client/containers/404Page/404Page.styles.css

```diff
@@ -1,3 +1,17 @@
-div {
-  color: #687808;
+.page404 {
+  text-align: center;
+}
+
+.page404 .btn {
+  background: #687808;
+  color: #ffff;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  height: 40px;
+  border: 0;
+}
+
+.page404 .link {
+  text-decoration: none;
```

Comment:
Color should be black for the text. And for the button to look like figma, can you add more padding-inline and some border radius?

Reply:
Done

Reply:
👍